### PR TITLE
ProxyDisclosure Scanner Add Ref

### PR DESCRIPTION
**Pull request by kingthorin**
Add an RFC reference to the proxy disclosure scanner & update manifest changes section.

Fixes zaproxy/zaproxy#3441

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties [hunk 1]
```diff
@@ -106,7 +106,7 @@ ascanalpha.ldapinjection.apachedirectoryservices.errormessages=Failed to identif
...
 ascanalpha.proxydisclosure.soln=Disable the 'TRACE' method on the proxy servers, as well as the origin web/application server.\nDisable the 'OPTIONS' method on the proxy servers, as well as the origin web/application server, if it is not required for other purposes, such as 'CORS' (Cross Origin Resource Sharing).\nConfigure the web and application servers with custom error pages, to prevent 'fingerprintable' product-specific error pages being leaked to the user in the event of HTTP errors, such as 'TRACK' requests for non-existent pages.\nConfigure all proxies, application servers, and web servers to prevent disclosure of the technology and version information in the 'Server' and 'X-Powered-By' HTTP response headers.\n
-ascanalpha.proxydisclosure.refs=???
+ascanalpha.proxydisclosure.refs=https://tools.ietf.org/html/rfc2068#section-14.31
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties]: Could use the more recent one https://tools.ietf.org/html/rfc7231#section-5.1.2 ?

**Comment by thc202** [comment]
Worth updating the changes?

**Comment by kingthorin** [comment]
Ya, will do

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties (hunk 1 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties]: Any comment on this suggestion?

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties]: Yup, I like that idea. Sorry I missed that comment last night. Change coming.

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties]: Done

**Comment by thc202** [comment]
Thank you!

**Pull request merged**